### ISSUE TITLE
still maintained/license?

### ISSUE THREAD

**Issue by richo** [Mortal/fbspinner#1]
I'm working on a framebuffer driver for an embedded linux device, and after scouring the internet your implementation seems to be the closest to what I'm looking to do. I didn't see a license in your repo, so I wanted to check in and see if you'd be comfortable adding one.

I also wanted to see if this project is still maintained? the part I want to extract is mostly the ioctl definitions and some of the blitting logic, but if you like I can extract that into a framebuffer-sys and then give you a PR porting this crate across to use it?

**Comment by richo** [comment]
Edit: I finally thought to check Cargo.toml and saw that it's licensed GPL2!

**Comment by Mortal** [comment]
If you want a different license, or feel like the license in Cargo.toml is not enough, please go ahead and propose one via a pull request.

The project is still maintained in the sense that I run it on my main pc, so I see the spinner animation when I boot my laptop every day - I can confirm that it still works on Linux 5.5.

**Comment by richo** [comment]
Gotcha, thanks! The license is fine (I was just looking for it in the wrong place). I asked about maintenance only because I've definitely had some times where I backported something only to have the original author be AWOL.

It turns out that the framebuffer crate has everything I needed, but thanks for publishing this- it was super helpful to have a working implementation to compare against while I was working out some bugs. I just opened a PR that migrates you to the framebuffer crate as well (and uses a trait to keep your FbWriter) but obviously feel no pressure at all to merge it.

Thankyou!
